lua/roslyn/sln/utils.lua

```diff
@@ -82,61 +113,91 @@ function M.root(buffer)
         }
     end
 
+
     if broad_search then
```

Comment:
I think later after the previous PR, we could probably think about this being the default value.

We can wait a bit with this though😄 IMO, this is the "ideal" behavior, but I just made it opt-in because of performance issues previously.